cluster/cluster.py

```diff
@@ -231,6 +232,7 @@ def cli():
 @click.option('--instance-type', type=str, default='t2.micro', help='Type of instance')
 @click.option('--master-nodes', default=1, type=int, help='Number of master nodes')
 @click.option('--worker-nodes', default=1, type=int, help='Number of worker nodes')
+@click.option('--etcd_s3_bucket', default='teapot-etcd-backup', help='S3 Bucket to backup ETCD to')
```

Comment:
Is `teapot-etcd-backup` the best default name? Maybe something like `k8s-etcd-backup` or `cluster-etcd-backup` is better?